pkg/mutation/assign_mutator.go

```diff
@@ -37,8 +39,41 @@ func (m *AssignMutator) Matches(obj runtime.Object, ns *corev1.Namespace) bool {
 	return matches
 }
 
+func (m *AssignMutator) gatherPathTests() ([]patht.Test, error) {
+	pts := m.assign.Spec.Parameters.PathTests
+	var pathTests []patht.Test
+	for _, pt := range pts {
+		p, err := parser.Parse(pt.SubPath)
+		if err != nil {
+			return nil, err
```

Comment:
I'm not certain the parser will include enough context as to which path failed - we might want to wrap and include that.

Reply:
SGTM, added a wrap